File: src/main/java/com/hohltier/decompiler/window/menu/edit/Edit.java
package com.hohltier.decompiler.window.menu.edit;

import com.hohltier.decompiler.window.utils.Language;

import javax.swing.*;
import java.awt.event.KeyEvent;

public class Edit extends JMenu {

    public Edit() {
        setText(Language.get("edit"));
        setMnemonic(KeyEvent.VK_E);

        add(new EditCopy());
        add(new EditSelectAll());
    }

}

File: src/main/kotlin/net/pryoscode/decompiler/window/menu/edit/Edit.kt
package net.pryoscode.decompiler.window.menu.edit

import net.pryoscode.decompiler.window.menu.edit.menus.Copy
import net.pryoscode.decompiler.window.menu.edit.menus.Find
import net.pryoscode.decompiler.window.menu.edit.menus.SelectAll
import java.awt.event.KeyEvent
import javax.swing.JMenu

class Edit : JMenu("Edit") {

    init {
        mnemonic = KeyEvent.VK_E
        add(Copy())
        add(SelectAll())
        add(Find())
    }

}